src/@batch-flask/ui/form/complex-form/complex-form.component.ts

```diff
@@ -268,4 +274,8 @@ export class ComplexFormComponent extends FormBase implements AfterViewInit, OnC
             multiUse: this.multiUse,
         };
     }
+
+    private focusFirstFocusableElement() {
+        this.formElement.nativeElement?.querySelector('[autofocus], button, input, textarea, select')?.focus()
+    }
```

Comment:
The existing autofocus directive will not work as expected when used on an element nested inside of `complex-form` because those elements seem to be injected/rendered in the DOM before they're visible/shown. We'll have to explicitly focus those elements after the nested form page is opened.